ecl/hql/hqlexpr.cpp

```diff
@@ -10476,10 +10476,13 @@ IHqlExpression *CHqlExternalDatasetCall::clone(HqlExprArray &newkids)
 
 //==============================================================================================================
 
-CHqlDelayedCall::CHqlDelayedCall(IHqlExpression * _param, ITypeInfo * _type, HqlExprArray &_ownedOperands) : CHqlExpressionWithType(no_call, _type, _ownedOperands), param(_param)
+CHqlDelayedCall::CHqlDelayedCall(IHqlExpression * _funcdef, ITypeInfo * _type, HqlExprArray &_ownedOperands) : CHqlExpressionWithType(no_call, _type, _ownedOperands), funcdef(_funcdef)
 {
-    infoFlags |= (param->getInfoFlags() & HEFalwaysInherit);
-    infoFlags2 |= (param->getInfoFlags2() & HEF2alwaysInherit);
+    IHqlExpression * body = funcdef;
```

Comment:
This is the main change - the rest are renaming a field, and improving the logging of transform stats when enabled.